### ISSUE TITLE
RTX3070 CUDA 11.1 CuDNN 8.0.5 not GPU available with tensorflow 2.5 (nightly version)

### ISSUE THREAD

**Issue by eperspicace** [tensorflow/tensorflow#45653]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10 Home
- TensorFlow installed from (source or binary): binary 
- TensorFlow version: tf-nightly-gpu==2.5.0.dev20201213
- Python version: 3.8.5
- Installed using virtualenv? pip? conda?: installed via anaconda (conda 4.9.2) using pip install tf-nightly-gpu
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: CUDA 11.1 / cuDNN 8.0.5
- GPU model and memory: Geforce RTX 3070 8 Go (Driver 460.79)

**Describe the problem**
I have encountered a problem when I tried to install tensorflow-gpu in my anaconda environment.

When I tested the availability of GPU after import of tensorflow, it seems that cusolver64_10.dll is missing and I cannot use my GPU with Tensorflow. From what I know, Tensorflow is not available for Cuda toolkit 11.1. So, should I try to compile Tensorflow or show I try older version of Cuda toolkit for example in virtual machine ?

**Provide the exact sequence of commands / steps that you executed before running into the problem**
(deeplearning-tf) C:\Users\Utilisateur>python
Python 3.8.5 (default, Sep  3 2020, 21:29:08) [MSC v.1916 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflos as tf
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'tensorflos'
>>> import tensorflow as tf
2020-12-14 12:30:59.774387: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
>>> print("Num GPUs Available: ", len(tf.config.experimental.list_physical_devices('GPU')))
2020-12-14 12:31:28.599495: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library nvcuda.dll
2020-12-14 12:31:28.631910: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1727] Found device 0 with properties:
pciBusID: 0000:2b:00.0 name: GeForce RTX 3070 computeCapability: 8.6
coreClock: 1.815GHz coreCount: 46 deviceMemorySize: 8.00GiB deviceMemoryBandwidth: 417.29GiB/s
2020-12-14 12:31:28.631996: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll
2020-12-14 12:31:29.003118: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublas64_11.dll
2020-12-14 12:31:29.003522: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublasLt64_11.dll
2020-12-14 12:31:29.266489: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cufft64_10.dll
2020-12-14 12:31:29.287591: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library curand64_10.dll
2020-12-14 12:31:29.288565: W tensorflow/stream_executor/platform/default/dso_loader.cc:60] Could not load dynamic library 'cusolver64_10.dll'; dlerror: cusolver64_10.dll not found
2020-12-14 12:31:29.462695: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusparse64_11.dll
2020-12-14 12:31:29.475294: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll
2020-12-14 12:31:29.475391: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1764] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
Num GPUs Available:  0

Thank you in advance for your help.
Best
Enrico

**Comment by bhack** [comment]
Yes please check https://github.com/tensorflow/tensorflow/issues/44697 https://github.com/tensorflow/tensorflow/issues/44291

**Comment by amahendrakar** [comment]
@eperspicace,
Could you please install CUDA 11 instead of CUDA 11.1 and check if you're facing the same issue?

Also as @bhack mentioned, please go through these similar issues [#44567](https://github.com/tensorflow/tensorflow/issues/44567#issuecomment-729504167), [#45258](https://github.com/tensorflow/tensorflow/issues/45258#issuecomment-739871133) for more information. Thanks!

**Comment by eperspicace** [comment]
@bhack, @amahendrakar 
thank you for your answers. I will uninstall CUDA11.1 and install CUDA11.0 instead. It should solve the problem of dll librairy. 
Best,
Enrico

**Comment by eperspicace** [comment]
Dear @bhack and @amahendrakar ,
I installed CUDA11.0 beside CUDA11.1 without uninstalling it (as mentioned in the link you provided). I also downloaded and unpacked cuDNN 8.0.4. Tensorflow was installed inside anaconda environment from pip tf-nightly-gpu numpy==1.19.3 command line. I was able to import tensorflow and test for GPU availability. It seems to work as I didn't get any error. Thank again for your help!
Regards,
Enrico
